sign/tbls/tbls_test.go

```diff
@@ -7,25 +7,45 @@ import (
 	"go.dedis.ch/kyber/v3/pairing/bn256"
 	"go.dedis.ch/kyber/v3/share"
 	"go.dedis.ch/kyber/v3/sign/bls"
+	"go.dedis.ch/kyber/v3/xof/blake2xb"
 )
 
 func TestTBLS(test *testing.T) {
-	var err error
-	msg := []byte("Hello threshold Boneh-Lynn-Shacham")
+	TBLSRoutine(test, []byte("Hello threshold Boneh-Lynn-Shacham"), 10)
+}
+
+func FuzzTBLS(f *testing.F) {
+	f.Fuzz(func(t *testing.T, msg []byte, n int) {
+		if (n < 1) || (n > 100) {
+			t.Skip("n must be between 1 and 100")
+		}
+		if (len(msg) < 1) || (len(msg) > 1000) {
+			t.Skip("msg must have byte length between 1 and 1000")
+		}
+		TBLSRoutine(t, msg, n)
+	})
+}
+
+func TBLSRoutine(test *testing.T, msg []byte, n int) {
 	suite := bn256.NewSuite()
```

Comment:
I think you also need to assign the below deterministic stream to the suite's stream to avoid having the Suite default to proper randomness.